database/certificate.go

```diff
@@ -29,6 +29,11 @@ func (db *DB) InsertCertificate(cert *certificate.Certificate) (int64, error) {
 		return -1, err
 	}
 
+	extkeyusageOID, err := json.Marshal(cert.X509v3Extensions.ExtendedKeyUsageOID)
```

Comment:
nit: consistent naming, if you're using `ExtKeyUsageOID` for the struct, you should use `extKeyUsageOID` here.